.dockerignore

```diff
@@ -1,4 +1,3 @@
-.git
```

Comment:
we will want to still ignore git here i think, any reason to have the `.git` info in the image?

Reply:
There was when it was needed to pull the log from it. I'll add it back, even though it's not doing any harm.